app/controllers/char_combats_controller.rb

```diff
@@ -0,0 +1,56 @@
+class CharCombatsController < ApplicationController
+
+  def index
```

Comment:
I'm guessing this is used for selecting combatants, if so it's better to be rest-full, that is use new. Since you are creating a new battle. If not, then what is this for?